frontend/components/organisms/QuestionForm/index.tsx

```diff
@@ -25,11 +30,33 @@ const QuestionForm = (): JSX.Element => {
         resolver: yupResolver(QuestionFormSchema),
     })
 
+    const router = useRouter()
+
+    const [createQuestion] = useMutation(CREATE_QUESTION)
+
     const onSubmit = (data: FormValues) => {
-        if (isObjectEmpty(errors)) {
-            //Replace with graphql post request
-            console.log(data)
-        }
+        const tags = data.tags.map((tag) => tag.id)
+
+        const newQuestion = createQuestion({
+            variables: {
+                title: data.title,
+                content: data.description,
+                is_public: true,
+                tags,
+                team_id: 1,
+            },
+        })
+
+        successNotify('Question Added Successfully')
+
+        newQuestion.then((data: any) => {
+            const slug = data.data.createQuestion.slug
+            const id = data.data.createQuestion.id
+            router.replace({
+                pathname: `/questions/${slug}`,
+                query: { id },
```

Comment:
please remove query field, it will show in the url, question id will be retrieved in question detail page